app/policies/admin/api_key_rubygem_scope_policy.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::ApiKeyRubygemScopePolicy < Admin::ApplicationPolicy
```

Comment:
git is failing to diff cleanly here. All of these are just copied over with the namespace added in front.